### DISCUSSION TITLE
rAudio 1 • 20210807: No more coverart displayed in playback window

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#459] [Fixed Issues]
rAudio 1 • 20210807
5.10.52-6-ARCH aarch64
Raspberry Pi 3 Model B Rev 1.2

After adding an album containing coverart, folder and thumb JPGs from the library to an empty playlist, the playlist window shows the "No cover" icon.

**Comment by ceejayemm** [comment]
I don't know if it helps, I had a similar problem with existing Web Radio logos after updating to 20210807,  I had to go to the Playback Display submenu and, with Coverart already selected, turn on the RAudio 'No Cover' option.  Seems counter intuitive but it got the Web Radio station logos back for me.

Chris

**Reply by klassikmann** [reply]
Thank you Chris,
The probIem appeared first at the webradios, but the albums are affected also. I could resolve the problem as you did but for a short period of time only. As I tried it again it looked as the trick only works once. I had to restore the latest backup then.

**Reply by AndyFx12** [reply]
Update-Branch contains a fix for Album-Coverart. Webradio-Coverart-issue is still present (and reported to rern).

Andy

**Comment by Rasb74** [comment]
Same for me 64bits version is perfect no noticed main bugs ! 
BUT I cannot now update it, the system request a new image file and I cannot find new 64bits image files…. 
does someone help me ?

**Reply by Roland-B** [reply]
@Rasb74 This is the latest 64-Bit release. which can be updated to 20210807.
https://github.com/rern/rAudio-1/releases/download/i20210716/rAudio-1-RPi64-20210716.img.xz

RolandB
